Assets/Scripts/Shootable/Shootable.cs

```diff
@@ -49,22 +49,28 @@ public void TakeDamage(int damage, Collider hitbox)
     {
         if (isDead) return;
 
-        float damage_mult = 1f;
+        float damageMult = 1f;
 
         foreach (Hitbox hit in Hitboxes)
         {
             if (hit.collider == hitbox)
             {
-                damage_mult = hit.multiplier;
+                damageMult = hit.multiplier;
             }
         }
 
-        int final_damage = (int)(damage * damage_mult);
+        int finalDamage = (int)(damage * damageMult);
 
-        currentHealth -= final_damage;
+        currentHealth -= finalDamage;
 
         if (currentHealth <= 0) Die();
     }
+    public int[] GetHealth()
+    {
+        int[] healthStatus = { currentHealth, furnitureSO.maxHealth };
+        return healthStatus;
+    }
+        
```

Comment:
## OPTIONAL CHANGE:
```suggestion
public (int current, int max) GetHealth()
{
        return (currentHealth, furnitureSO.maxHealth);
}
```
This may be clearer as a [`tuple`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples). You could return an `(int, int)` tuple with named elements.